sql-plugin/src/main/scala/com/nvidia/spark/rapids/conditionalExpressions.scala

```diff
@@ -28,23 +28,9 @@ abstract class GpuConditionalExpression extends ComplexTypeMergingExpression wit
   private def computePredicate(
```

Comment:
nit: This gets called in just two places in the code, and they are in this same class.  Could we just replace the entire method with `GpuExpressionsUtils.columnarEvalToColumn`

Reply:
Done.
Verified the Scala tests on my node and they passed.
Triggered pre-merge build to run the integration tests.

I still have a little concern of using the `GpuExpressionsUtils.columnarEvalToColumn` here, because it implicitly allows a `GpuScalar` as the result of the predicate expression, but the original code did not.

Reply:
Yes, but if a GpuScalar shows up it will do the right thing, so it should be more robust than the old code.